### PR TITLE
add ci and support julia 1.7

### PR DESCRIPTION
**Pull request by jw3126**

### CODE REVIEW

File: test/meshes.jl [hunk 1]
```diff
@@ -10,7 +10,7 @@ using GeometryTypes: slice
         (HyperRectangle{3,Float32}(Vec3f0(baselen), Vec3f0(baselen, baselen, dirlen)), RGBA(0f0,0f0,1f0,1f0))
     ]
 
-    am = map(GLNormalMesh, a)
+    am = [GLNormalMesh(x) for x in a]
```
Comment (jw3126) [inline test/meshes.jl:13]: The design of GeometryTypes is such that `GLNormalMesh(x) :: GLNormalMesh` does NOT hold. On recent Julia version, this causes map to fail.

**Comment by jw3126** [comment]
@JeffreySarnoff could you merge this and tag a release?

**Comment by JeffreySarnoff** [comment]
yes .. am doing so.

**Comment by jw3126** [comment]
wow that was quick! Thanks a lot @JeffreySarnoff !

**Comment by JeffreySarnoff** [comment]
happy to have been present -- _next time we may get it all done_ 😀

After merging the PR and bumping the version, it was time to register the new version by commenting `@JuliaRegistrator register()`; which I did.  GitHub said
> Error while trying to register: Register Failed
@JeffreySarnoff, it looks like you are not a publicly listed member/owner in the parent organization (JuliaGeometry). If you are a member/owner, you will need to change your membership to public. 

@admin if this is intended, please start the registration; otherwise, how do I fix it?
`GitHub Help` provided me no help (I may have missed something).

**Comment by jw3126** [comment]
@JeffreySarnoff I had this problem before. For me this guide 
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership 
worked fine.

**Comment by JeffreySarnoff** [comment]
great help .. I had been looking to  a different part of GeometryTypes .. problem solved

**Pull request merged**